modules/Bio/EnsEMBL/Compara/PipeConfig/EG/ProteinTrees_conf.pm

```diff
@@ -73,95 +72,40 @@ sub default_options {
-        'cdhit_identity_threshold' => 0.99,
-
-    # clustering parameters:
-
-    # tree building parameters:
-        'use_quick_tree_break'      => 1,
-
-    # alignment filtering options
-
-    # species tree reconciliation
-
     # homology_dnds parameters:
         # used by 'homology_dNdS'
         'taxlevels'                 => [],  # this is the default default
         'taxlevels_fungi'           => ['Botryosphaeriales', 'Calosphaeriales', 'Capnodiales', 'Chaetothyriales', 'Dothideales', 'Erysiphales', 'Eurotiales', 'Glomerellales', 'Helotiales', 'Hypocreales', 'Microascales', 'Onygenales', 'Ophiostomatales', 'Orbiliales', 'Pleosporales', 'Pneumocystidales', 'Saccharomycetales', 'Sordariales', 'Verrucariales', 'Xylariales', 'Venturiales', 'Agaricales', 'Atheliales', 'Boletales', 'Cantharellales', 'Corticiales', 'Dacrymycetales', 'Geastrales', 'Georgefischeriales', 'Gloeophyllales', 'Jaapiales', 'Malasseziales', 'Mixiales', 'Polyporales', 'Russulales', 'Sebacinales', 'Sporidiobolales', 'Tremellales', 'Ustilaginales', 'Wallemiales', 'Cryptomycota', 'Glomerales ', 'Rhizophydiales'],
         'taxlevels_metazoa'         => ['Drosophila' ,'Hymenoptera', 'Nematoda'],
-        'taxlevels_plants'          => ['Liliopsida', 'eudicotyledons', 'Chlorophyta'],
         'taxlevels_protists'        => ['Alveolata', 'Amoebozoa', 'Choanoflagellida', 'Cryptophyta', 'Fornicata', 'Haptophyceae', 'Kinetoplastida', 'Rhizaria', 'Rhodophyta', 'Stramenopiles'],
         'taxlevels_vb'              => ['Calyptratae', 'Culicidae'],
 
-    # mapping parameters:
-        'do_stable_id_mapping'      => 1,
-        'do_treefam_xref'           => 1,
-        # The TreeFam release to map to
-        'tf_release'                => '9_69',
-
-    # executable locations:
-
     # hive_capacity values for some analyses:
-        'reuse_capacity'            =>   3,
-        'blast_factory_capacity'    =>  50,
         'blastp_capacity'           => 200,
         'blastpu_capacity'          => 100,
-        'mcoffee_capacity'          => 200,
         'split_genes_capacity'      => 200,
-        'alignment_filtering_capacity'  => 200,
-        'prottest_capacity'         => 200,
+        'cluster_tagging_capacity'  => 200,
+        'homology_dNdS_capacity'    => 200,
         'treebest_capacity'         => 200,
-        'raxml_capacity'            => 200,
-        'examl_capacity'            => 400,
-        'notung_capacity'           => 200,
         'ortho_tree_capacity'       => 200,
         'quick_tree_break_capacity' => 100,
-        'build_hmm_capacity'        => 200,
-        'ktreedist_capacity'        => 150,
         'goc_capacity'              => 200,
-        'goc_stats_capacity'        => 15,
-        'genesetQC_capacity'        => 100,
+        'goc_stats_capacity'        =>  15,
         'other_paralogs_capacity'   => 100,
-        'homology_dNdS_capacity'    => 200,
+        'mcoffee_short_capacity'    => 200,
```

Comment:
Where did this number come from?

Reply:
I tracked `mcoffee_capacity` was renamed to `mcoffee_short_capacity` at some point but this change wasn't applied to this list (you can see a little bit earlier in the diff that the former name has been deleted). Sorry for the confusion, I decided to reorder the hash to follow the same order as in the base class: I find it easier to track overwritten parameters if both hashes follow the same order.